lib/graphql/query/null_context.rb

```diff
@@ -21,9 +21,13 @@ def initialize
         )
       end
 
+      def [](key); end
```

Comment:
Is this a reasonable thing to do? I feel like a null context should be able to be accessible here and just return nil? Same with the instance method?

Reply:
This follows along with the GraphQL Context interface so I think it is reasonable to give it similar functionality. `https://www.rubydoc.info/github/rmosolgo/graphql-ruby/GraphQL/Query/Context`